static/main.js

```diff
@@ -57,28 +57,34 @@
 (function() {
   var games = document.getElementsByTagName('dd');
   var tags = document.getElementsByClassName('tag');
+  var activeTag;
 
   for (var i = 0, l = tags.length; i < l; i += 1) {
     tags[i].addEventListener('click', onclick);
   }
 
   function onclick(e) {
     var t = e.target.hasAttribute('data-name') ? e.target : e.target.parentNode;
+    var curTag = t.getAttribute('data-name');
     var parent;
     var game, gameTags;
 
-    if (!hasClass(document.body, 'tags-active')) {
-      document.body.className += ' tags-active';
+    if (curTag === activeTag) {
+      activeTag = false;
+      document.body.className = '';
```

Comment:
Not sure if that's good to do, since some browser extensions tend to assign classes on `document.body`. On the other hand, that's their problem, but we can do `document.body.className = document.body.className.replace(/ ?\btags-active\b/, '')` (not sure this regexp is correct, haha).